frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -46,7 +46,7 @@ export const createNoobaaPayload = (): Payload => {
       plural,
     },
     payload: {
-      apiVersion,
+      apiVersion: apiVersionForModel(NooBaaSystemModel),
```

Comment:
```suggestion
      apiVersion,
```
Do this or either don't destructure it at line 39.

Reply:
Check line L44, the destructured value used over there.
And apiVersion is not the correct value for model.